tests/e2e/helpers/api/api-keys.ts

```diff
@@ -0,0 +1,128 @@
+import { t } from 'testcafe';
+import * as request from 'supertest';
+import { AddNewDatabaseParameters } from '../../pageObjects/add-redis-database-page';
+import { apiUrl } from '../../helpers/conf';
+import {
+    HashKeyParameters,
+    ListKeyParameters,
+    SetKeyParameters,
+    SortedSetKeyParameters,
+    StreamKeyParameters
+} from '../../pageObjects/browser-page';
+import { getDatabaseByName } from './api-database';
+
+const endpoint = apiUrl;
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'; // lgtm[js/disabling-certificate-validation]
```

Comment:
Don't you already use it in past? Maybe you come to move it to a common place and import it?

Reply:
moved to conf.ts
thanks!